rules/QW0016.md

```diff
@@ -0,0 +1,34 @@
+﻿# QW0016: Prefer regular over positional properties 
+
+With the release of C# 10, [`record`](https://learn.microsoft.com/dotnet/csharp/language-reference/builtin-types/record)'s
+have been introduced. One of the advocated advantages is the possibility to use
+a so called primary constructor which allows to define a full data structure in
+a single line:
+
+``` C#
+record MyRecord(string Message, string Other);
+```
+
+However powerful, for (public) API's it turns out to be not that practical:
+Defining attributes, having numerous (Let say 3 or more) properties, or
```

Comment:
This sentence doesn't feel right, I would suggest splitting the "numerous properties" part into a separate sentence.
Suggestion: "Defining attributes or adding documentation turns out to be counterintuitive and cumbersome." "Having numerous properties (let's say 3 or more) also makes it less readable."